python-avd/pyavd/api/ip_addressing/__init__.py

```diff
@@ -331,3 +331,27 @@ def evpn_underlay_l2_multicast_group(
         """Return IP address to be used for EVPN underlay L2 multicast group."""
         offset = vlan_id - 1 + underlay_l2_multicast_group_ipv4_pool_offset
         return get_ip_from_pool(underlay_l2_multicast_group_ipv4_pool, 32, offset, 0)
+
+    def wan_ha_ip_local(self) -> str:
```

Comment:
For consistency with the other methods

```suggestion
    def wan_ha_ip(self) -> str:
```